src/internal/menu/StartMenu.tsx

```diff
@@ -95,13 +96,40 @@ export function StartMenuComponent({header, mode, open}: {header: AppHeader, mod
     header.toggleDarkTheme((activeTab) ? true : false);
   }, [header, activeTab]);
 
+
+  React.useEffect(() => {
+    // add short cut button to current session card to navbar in header
+    let currentSessionNav = header.rightMenu.parentElement.querySelector('.current-session') as HTMLUListElement;
+
+    // add menu only once
+    if (!currentSessionNav) {
+      // TODO once the phovea header is using React we can switch to `Nav` from react bootstrap
+      currentSessionNav = header.rightMenu.ownerDocument.createElement('ul');
+      currentSessionNav.classList.add('navbar-nav', 'navbar-right', 'current-session');
+
+      ReactDOM.render(<Nav.Link><i className="fas fa-history mr-2"></i>Current Analysis Session</Nav.Link>, currentSessionNav);
+
+      currentSessionNav.onclick = (event) => {
+        event.preventDefault();
+        setActiveTab(tabs[1]); // TODO: find better way to identify the tabs
```

Comment:
I also don't have a better idea right now. We have to refactor/extract the tabs anyway to somewhere else  in the future. Maybe then we have a better idea.